### PR TITLE
Remove outdated version of salt-formula-galera from 'linux' formula

### PR DESCRIPTION
**Pull request by dis-xcom**
Fix of the bug#PROD-7972

Avoid downgrading 'salt-formula-galera' package to the hardcoded outdated version.

### CODE REVIEW

File: classes/system/salt/master/pkg.yml [hunk 1]
```diff
@@ -11,7 +11,3 @@ parameters:
           source: "deb [arch=amd64] http://apt.tcpcloud.eu/nightly/ trusty tcp-salt"
           architectures: amd64
           key_url: "http://apt.tcpcloud.eu/public.gpg"
-      package:
-        salt-formula-galera:
```
Comment (TimurNurlygayanov) [inline classes/system/salt/master/pkg.yml]: Will we install this package if we remove completely these lines? Is it better to remove only the line #16?

File: nodes/control/cfg01.mk22-lab-advanced.local.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ classes:
 - system.openstack.common.mk22_lab_advanced
 parameters:
   _param:
-    reclass_data_repository: "git@github.com:Mirantis/mk-lab-salt-model.git"
+    reclass_data_repository: "https://github.com/dis-xcom/mk-lab-salt-model.git"
```
Comment (elemoine) [inline nodes/control/cfg01.mk22-lab-advanced.local.yml]: You will want to remove this and squash your commits I guess.

Comment (dis-xcom) [inline nodes/control/cfg01.mk22-lab-advanced.local.yml]: Yes, it was for check the change on a deploy. Removed.

File: classes/system/salt/master/pkg.yml (hunk 1 above)
Comment (cznewt) [inline classes/system/salt/master/pkg.yml]: this was backward compatibility pin, remove it to get it working with mos

**Pull request merged**